spring-ai-core/src/main/java/org/springframework/ai/model/TranscriptionModel.java

```diff
@@ -0,0 +1,22 @@
+package org.springframework.ai.model;
```

Comment:
Should this interface be in the package `org.springframework.ai.model.audio.transcription`?